src/psyclone/tests/dependency_test.py

```diff
@@ -349,6 +349,43 @@ def test_dynamo():
             "nlayers: READ, undf_w1: READ, undf_w2: READ, undf_w3: READ")
 
 
+def test_dynamo_args():
```

Comment:
We're gradually renaming everything to "lfric" so "test_lfric_kern_cma_args" please.

Reply:
Good point, done.